spring-boot/pom.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>infinispan-simple-tutorials-spring-boot</artifactId>
+    <parent>
+        <relativePath>../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+        <groupId>org.infinispan.tutorial.simple</groupId>
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+
+    <name>Infinispan Simple Tutorials: Spring Boot</name>
+
+    <properties>
+        <spring-boot.version>1.4.0.RELEASE</spring-boot.version>
+        <infinispan.starters.version>1.0.0-SNAPSHOT</infinispan.starters.version>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-parent</artifactId>
+                <version>${spring-boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>inifinispan-spring-boot-starter</artifactId>
+            <version>${infinispan.starters.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.infinispan</groupId>
+                    <artifactId>infinispan-core</artifactId>
+                </exclusion>
```

Comment:
This is  weird. So the starter brings the embedded mode and you have to remove it to add the client-side mode. Everything in Spring Boot works the other way around: the starter brings the client/server support and if you want to embed (which arguably is going to bring more dependencies), then you should add it. 

The naming doesn't help, `infinispan-core` makes me think of something I should always have. 

Is there a reason you are doing this the other way around. Having to exclude something to add the lightweight option makes my eyes bleed.